src/all/batoto/src/eu/kanade/tachiyomi/extension/all/batoto/BatoTo.kt

```diff
@@ -384,12 +384,20 @@ open class BatoTo(
         val chapter = SChapter.create()
         val urlElement = element.select("a.chapt")
         val group = element.select("div.extra > a:not(.ps-3)").text()
+        val user = element.select("div.extra > a.ps-3").text()
         val time = element.select("div.extra > i.ps-3").text()
         chapter.setUrlWithoutDomain(urlElement.attr("href"))
         chapter.name = urlElement.text()
         if (group != "") {
             chapter.scanlator = group
         }
+        if (group.isNullOrBlank()) {
+            chapter.scanlator = user
+        }
+        if (user.isNullOrBlank()) {
+            chapter.scanlator = "Unknown"
+        }
+
```

Comment:
use else if

Reply:
I tried using `else if` on both, it didn't work... (not in the third commit because I'm a dumbass sry)
I tried using `else` instead of `else if` in the `user.isNullOrBlank` and it also didn't work. (both builds were successful though)
The most recent commit was the only way I found to make everything work... although, if there's a better way, please do tell me and I'll try to fix it again